neofetch

```diff
@@ -3202,6 +3202,49 @@ END
             # Default fallback font hardcoded in terminal-preferences.c
             [[ -z "$term_font" ]] && term_font="Monospace 12"
         ;;
+
+        "conemu-"*)
+            local ce_arg_list
+            local ce_arg_idx
+            local ce_conf
+
+            # Could have used `eval set -- "$ConEmuArgs"` instead for arg parsing but eval is evil...
+            readarray -t ce_arg_list < <(xargs -n1 printf "%s\n" <<< "${ConEmuArgs-}")
+
+            for ce_arg_idx in "${!ce_arg_list[@]}"; do
+                # Search for "-LoadCfgFile" arg
+                [[ "${ce_arg_list[$ce_arg_idx]}" != -LoadCfgFile ]] && continue
+
+                # Conf path is the next arg
+                ((++ce_arg_idx))
+                ce_conf="${ce_arg_list[$ce_arg_idx]}"
+                break
+            done
+
+            # https://conemu.github.io/en/ConEmuXml.html#search-sequence
+            local ce_seq=(
+                "$ce_conf"
+                "$ConEmuDir\ConEmu.xml"
+                "$ConEmuDir\.ConEmu.xml"
+                "$ConEmuBaseDir\ConEmu.xml"
+                "$ConEmuBaseDir\.ConEmu.xml"
+                "$APPDATA\ConEmu.xml"
+                "$APPDATA\.ConEmu.xml"
+            )
+
+            for ce_conf in "${ce_seq[@]}"; do
+                # Search for first conf file available
+                [[ ! -f "$ce_conf" ]] && continue
+
+                # Very basic XML parsing
+                term_font="$(awk '/name="FontName"/ && match($0, /data="([^"]*)"/) {print substr($0, RSTART+6, RLENGTH-7)}' "$ce_conf")"
+                break
+            done
+
+            # Null-terminated contents in /proc/registry files triggers a Bash warning. Use read instead
+            [[ -z "$term_font" ]] && \
+                read -r term_font < /proc/registry/HKEY_CURRENT_USER/Software/ConEmu/.Vanilla/FontName
```

Comment:
Another option is: 

```
term_font=${term_font:-$(< /proc/registry/HKEY_CURRENT_USER/Software/Conemu/.Vanilla/FontName)}
```

The above test can be omitted entirely if this works too.

Reply:
I already tried `$(< file)` but it gives me this warning with `/proc/registry` files as their content is null-terminated :

```
C:\cygwin64\bin\bash: warning: command substitution: ignored null byte in input
```

```
$ hexdump -C < /proc/registry/HKEY_CURRENT_USER/Software/Conemu/.Vanilla/FontName
00000000  43 6f 75 72 69 65 72 00                           |Courier.|
```

As far as I know, either we keep the test + `read` or sanitize with `awk`, `sed`, `tr` or the like.